BCP195bis/draft-ietf-uta-rfc7525bis.md

```diff
@@ -1114,9 +1114,21 @@ on the normative changes.
 
 <cref>Note to RFC Editor: please remove before publication.</cref>
 
+## draft-ietf-uta-rfc7525bis-10
+
+* Addressed IESG feedback, ARTART review by Cullen Jennings, and TSVART review by Magnus Westerlund.
+* Improved the rationale for still recommending TLS 1.2.
+* Specified TLS 1.3 as a MUST for new transport protocols and a SHOULD for new application protocols.
+* Clarified TLS-only vs. dynamic upgrade for non-HTTP protocols.
+* Clarified distinction between implementation and deployment.
+* Noted that these recommendations might apply to QUIC, but that QUIC is out of scope.
```

Comment:
```suggestion
* Clarified applicability to QUIC.
```

I think the text is problematic (if they "might apply", how is the protocol out of scope). So let's be higher level here, and wrangle the exact text elsewhere.